contenttype/utils.go

```diff
@@ -33,6 +33,11 @@ func IsStringContentType(contentType string) bool {
 	return isContentType(contentType, "application/xml")
 }
 
+// IsBinaryContentType determines if content type is byte[]
+func IsBinaryContentType(contentType string) bool {
+	return isContentType(contentType, "application/octet-stream")
```

Comment:
This is fine for now, later we will add proper content type detection

Reply:
Correct. We should use a richer detection using MimeType hierarchy. This issue is about that: https://github.com/dapr/components-contrib/issues/635